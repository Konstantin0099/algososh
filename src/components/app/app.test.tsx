
import React from "react";
import App from "./app";
import { render, screen } from "@testing-library/react";
import renderer from "react-test-renderer";
import { Link } from "react-router-dom";
import styles from "./main-page.module.css";


describe('todos reducer', () => {
  
  it("!!!!!!!!!!!!!!!renders Link", () => {
  render(<App />);
  const linkElement1 = screen.getByText("им. Фибоначчи");
  const tree = renderer
  .create( <div className={`${styles.card} ${styles.string}`} />)
  .toJSON();
  expect(tree).toMatchSnapshot();
  expect(linkElement1).toBeInTheDocument();
});


})
