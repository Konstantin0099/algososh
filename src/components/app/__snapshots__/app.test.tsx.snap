// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`todos reducer !!!!!!!!!!!!!!!renders Link 1`] = `
<div
  className="card string"
/>
`;
